src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ProjectTypeCapabilities.cs

```diff
@@ -29,11 +29,15 @@ internal class ProjectTypeCapabilities
         /// <summary>
         ///     Represents C#'s set of capabilities that are always present.
         /// </summary>
-        public const string CSharp = Default + "; " + ProjectCapability.CSharp;
+        public const string CSharp = Default + "; " +
+                                     ProjectCapability.CSharp + "; " + 
+                                     ProjectCapabilities.SharedImports;
 
         /// <summary>
         ///     Represents Visual Basic's set of capabilities that are always present.
         /// </summary>
-        public const string VisualBasic = Default + "; " + ProjectCapability.VisualBasic;
+        public const string VisualBasic = Default + "; " + 
+                                          ProjectCapability.VisualBasic + "; " + 
+                                          ProjectCapabilities.SharedImports;
```

Comment:
This doesn't apply to F# projects?

Reply:
F# doesn't support shared projects. It requires language support.